Modules/Core/Common/ITKKWStyleOverwrite.txt

```diff
@@ -33,3 +33,14 @@ itkMultiThreaderBase.h Comments Disable
 itkSingleton\.cxx Namespace Disable
 itkExceptionObject\.h IfNDefDefine Disable
 itkConceptChecking\.h Template Disable
+itkCommonEnums\.h Comments Disable
```

Comment:
Thanks for your great work! Do I understand correctly that `Comments Disable` might as well just be applied to each and every source file? Because it just isn't a useful warning anyway?

Reply:
@thewtex and @blowekamp might have some comments about this.

Reply:
The use of `\class ClassName` was a style decision, I believe. We could add these `Comments Disable` for the enum class files. Or, we could change our style, but we probably want to do it consistently throughout and then disable the option globally with the KWStyle rules: https://github.com/InsightSoftwareConsortium/ITK/blob/68619448ee9a6303f0b5958c4c7700c1fd5f78b3/Utilities/KWStyle/ITK.kws.xml#L12